plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraType.java

```diff
@@ -91,11 +96,11 @@
 import static java.lang.Float.intBitsToFloat;
 import static java.lang.Math.toIntExact;
 import static java.lang.String.format;
+import static java.lang.System.arraycopy;
 import static java.util.Objects.requireNonNull;
 
 public class CassandraType
 {
-    @SuppressWarnings("unused")
     private final TypeManager typeManager;
```

Comment:
Please define `Type ipAddressType` and initialize it in the constructor. Then, remove this `typeManager`.